frontend/test/metabase/scenarios/question/snippets.cy.spec.js

```diff
@@ -1,59 +1,96 @@
 import { signInAsNormalUser, restore, modal } from "__support__/cypress";
 
+function _clearAndRecursivelyTypeUsingLabel(label, string) {
+  // Cypress throws an error when trying to type an empty string
+  if (!string) {
+    return;
+  }
```

Comment:
I don't think you need this. `for (const char of "")` won't run the loop body at all since there are no characters.

Also, I think it would be the right move to let Cypress error if someone tries to type `""`. IMO it's better to let the test author know there's an issue.

Reply:
Good point(s).
Will remove it.